_opensearch/rest-api/nodes-apis/nodes-reload-secure.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Nodes reload secure settings
+parent: Nodes APIs
+grand_parent: REST API reference
+nav_order: 50
+---
+
+# Nodes reload secure settings
+
+The nodes reload secure settings endpoint allows you to change secure settings on a node and reload the secure settings without restarting the node.
+
+## Path and HTTP methods
+
+```
+POST _nodes/reload_secure_settings
+POST _nodes/<node id>/reload_secure_settings
+```
+
+## Path parameter
+
+You can include the following optional path parameter in your request:
+
+Parameter | Type | Description
+:--- | :--- | :---
+node_id | String | The names of the nodes to reload.
```

Comment:
```suggestion
node_id | String | nodeId | String | A comma-separated list of nodeIds to filter results. Supports [node filters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/nodes-apis/index/#node-filters). Defaults to `_all`.
```

Reply:
I don't know if we've reached consensus on how to format parameters in angle brackets, but other nodes APIs use <nodeId>. We can change that, but we need to make it consistent across all nodes APIs.